vulnerabilities/tests/test_fix_api.py

```diff
@@ -258,6 +271,12 @@ def test_api_with_single_vulnerability_and_vulnerable_package(self):
             ],
         }
 
+    def test_api_with_all_vulnerable_packages(self):
+        response = self.csrf_client.get(f"/api/packages/all", format="json").data
+        assert len(response) == 11
+        for package in self.vulnerable_packages:
+            assert package in response
```

Comment:
I would be preferable to directly assert the content of response.

Reply:
Done!

Reply:
Where? You are still testing the len() of the content, not the content itself.

Reply:
@tdruez here

Reply:
Ok, but as a reader of the test, it's hard to figure out the content of the variable.

If I'm not mistaken, the content of the response is something like:
```
[
   {"purl": purl1},
   {"purl": purl2},
   ....
]
```

Could you confirm?

Reply:
I have tried to make the tests more readable, please check if it looks good

Reply:
It does look better. 
Now we have a problem with this data structure.
Returning a list of dicts instead of a simple list of purls make the response content larger than is needs to be.
In the context of returning hundred of thousands of records, this is not ideal.

Reply:
Done!